orchestrate/clusters/i3_test.yml

```diff
@@ -0,0 +1,6 @@
+cloud_provider: aws
+cluster_name: i3-test
+cpu:
+  instance_type: i3.large
+  max_nodes: 2
+  min_nodes: 1
```

Comment:
max nodes can equal min nodes since we haven't implemented autoscaling yet

Reply:
Why do we have max and min nodes as options then right now? Is it for when we implement it?